tests/study_tests/test_optimize.py

```diff
@@ -71,22 +71,22 @@ def test_run_trial_automatically_fail(storage_mode: str, caplog: LogCaptureFixtu
         assert "The value nan is not acceptable." in caplog.text
 
         caplog.clear()
-        frozen_trial = _optimize._run_trial(study, lambda _: None, catch=())  # type: ignore
```

Comment:
Not sure if this is a bug in mypy, but `type: ignore` is considered unused. `# type: ignore[<code>]` may work.

Reply:
Hmm, I could not reproduce this in my local environment: mypy didn't complain about `type: ignore`. Instad, mypy says:

```
tests/samplers_tests/test_grid.py:42: error: Unused "type: ignore" comment
tests/samplers_tests/test_grid.py:51: error: Unused "type: ignore" comment
tests/samplers_tests/test_grid.py:57: error: Unused "type: ignore" comment
tests/samplers_tests/test_grid.py:69: error: Unused "type: ignore" comment
```

Reply:
@nzw0301 Thanks for the comment! Let me clarify what I meant.

The second argument of `_run_trial` should return `float` or `Sequence[float]` but `lambda _: None` returns `None`. `# type: ignore` is **used** to ignore this error but mypy thinks it's unused.